.gitignore

```diff
@@ -32,9 +32,9 @@ _testmain.go
 /http/web_ui/*.*
 /http/web_ui/**/*.*
 
-# Vault-specific
+# OpenBao-specific
 example.hcl
-example.vault.d
```

Comment:
I'd probably leave this as `vault.d` as well, for convenience. :-)

Reply:
fixed